packages/open-truss/src/configuration/engine-v1/RenderConfig.tsx

```diff
@@ -28,13 +35,30 @@ export function RenderConfig({
     // We likely want to render something nicer eventually.
     throw result.error
   }
-  const globalContext: GlobalContext = { config: result.data, COMPONENTS }
+  const FrameWrapper = getComponent(
+    config.frameWrapper ?? 'OTDefaultFrameWrapper',
+    { ...COMPONENTS, OTDefaultFrameWrapper },
+  )
+  const globalContext: GlobalContext = {
+    config: result.data,
+    COMPONENTS,
+    FrameWrapper,
+  }
 
   return (
     <>
-      {config.frames.map((frame, i) => (
-        <Frame key={i} frame={frame} globalContext={globalContext} />
-      ))}
+      {config.frames.map((frame, i) => {
```

Comment:
Hmm, this code is largely the same as the code in `Frame`. I wonder if there is a way to cast `config` as `frame` so we can just render the `Frame` and use much of the same code. 🤔

Reply:
Yes I think so! This exact thing is in my todos